extensions/dicom-segmentation/src/loadSegmentation.js

```diff
@@ -44,6 +44,11 @@ export default async function loadSegmentation(
       reject(error);
     }
 
+    if (!results) {
+      segDisplaySet.isLoaded = false;
+      return reject(new Error('Failed to parse segmentation.'));
```

Comment:
@JamesAPetts maybe we should create an issue to avoid undefined values coming from dcmjs generateToolState adapter for seg?

Reply:
You can test here: http://localhost:3000/projects/idc-tcia/locations/us/datasets/issues/dicomStores/issue-2-kits/study/1.3.6.1.4.1.14519.5.2.1.6919.4624.262333270527308026207113408715

Just switch to the first segmentation in the dropdown.